packages/prosemirror-lwdita-backend/README.md

```diff
@@ -0,0 +1,27 @@
+# prosemirror-lwdita-backend
+API module for prosemirror lwdita
+
+This is a standalone module with it's own http server.
+The API offer integration with Source control without user interaction to make contribution from prosemirror-lwdita possible.
+It offers the functionality to take changes from prosemirro-lwdita and create contributions (currently only GitHub is supported).
+
+# Setup
+install dependencies
+```shell
+$ yarn install
+```
+
+
+# Usage
+Build the code
+```shell
+$ yarn run build
+```
+
+Start the server:
+```shell
+$ yarn run start
+```
+
+# API documenting
```

Comment:
@marmoure Typo & grammar, please fix.

Reply:
fixed in fe0ad59982f00b312827a75cb369cd835786c518